packages/components/src/Modal/InputEditModal/_docs/InputEditModal.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react"
+import { Decorator, Meta, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic"
+import { ModalDescription } from "~components/Modal/GenericModal/subcomponents/ModalDescription"
```

Comment:
Can either do this or a relative path.
```suggestion
import { ModalDescription } from "~components/Modal"
```